app/controllers/concerns/galtersufia/contact_form_controller_behavior.rb

```diff
@@ -7,6 +7,25 @@ module ContactFormControllerBehavior
     extend ActiveSupport::Concern
     include Sufia::ContactFormControllerBehavior
 
+    included do
+      before_action :spam?, only: [:create]
+    end
+
+    def spam?
+      spam_msg = "potential spam detected for IP #{request.env['REMOTE_ADDR']}."
+      time_to_comment = Time.now - Time.parse(session['antispam_timestamp'])
+      if params[:contact_form][:customerDetail].present?
```

Comment:
so a script/bot is supposed to fill this section of the form out? interesting solution.

Reply:
All this is copied from what works for us on the website. Yes, the field is invisible to the user but visible to the bot that doesn't care about CSS-based properties.